### PR TITLE
Better links

### LINKED ISSUE #749: Add rel="noopener noreferrer" to external links
[More info](https://web.dev/external-anchors-use-rel-noopener/).

### PR DESCRIPTION
**Pull request by DannySS123**
Added rel="noopener" to external links to make them safer
Now online event links are opened in another tab just like other external links

Closes #749

### CODE REVIEW

**Review by Tschonti — DISMISSED** [review]

File: views/group/index.pug [hunk 1]
```diff
@@ -43,7 +43,7 @@ block content
               //- Heroicon name: external-link
               svg(xmlns='http://www.w3.org/2000/svg' alt="Külső link" aria-label="Külső link" fill='none' viewbox='0 0 24 24' stroke='currentColor' class='w-6 h-6')
                 path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14')
-              a(href=group.link class='text-lg sm:text-xl')= group.link
+              a(href=group.link class='text-lg sm:text-xl' target='_blank' rel='noopener')= group.link
```
Comment (triszt4n) [inline views/group/index.pug:46]: Maybe we could put a `noreferrer` to these external links too. [(source of tip)](https://clever-solution.com/everything-you-need-to-know-about-rel-noopener-noreferrer-tags-purpose-benefits-and-seo-impact/)

File: views/group/show.pug [hunk 2]
```diff
@@ -99,7 +99,7 @@ block content
               //- Heroicon name: external-link
               svg(xmlns='http://www.w3.org/2000/svg' alt="Külső link" aria-label="Külső link" fill='none' viewbox='0 0 24 24' stroke='currentColor' class='w-6 h-6')
                 path(stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14')
-              a(href=group.link class='text-lg sm:text-xl')= group.link
+              a(href=group.link class='text-lg sm:text-xl' target='_blank' rel='noopener')= group.link
```
Comment (triszt4n) [inline views/group/show.pug:102]: Same here

**Review by triszt4n — COMMENTED** [review]
Noice. Had a question here and there

File: views/group/index.pug (hunk 1 above)
Comment (OmTheTurtle) [reply views/group/index.pug:46]: We should have `noopener noreferrer` added to all external links.

**Review by OmTheTurtle — APPROVED** [review]

**Pull request merged**